crates/starknet-devnet/tests/test_abort_blocks.rs

```diff
@@ -58,23 +59,30 @@ mod abort_blocks_tests {
                 .await
                 .expect("Could not start Devnet");
 
-        let first_block_hash = devnet.create_block().await.unwrap();
-        let second_block_hash = devnet.create_block().await.unwrap();
+        let genesis_block =
+            devnet.json_rpc_client.get_block_with_tx_hashes(BlockId::Tag(BlockTag::Latest)).await;
+        let genesis_block_hash = match genesis_block {
+            Ok(MaybePendingBlockWithTxHashes::Block(b)) => b.block_hash,
+            _ => panic!("Unexpected resp: {genesis_block:?}"),
+        };
 
-        let aborted_blocks = abort_blocks(&devnet, &second_block_hash).await;
-        assert_eq!(aborted_blocks, vec![second_block_hash]);
+        let new_block_hash = devnet.create_block().await.unwrap();
+        let aborted_blocks = abort_blocks(&devnet, &new_block_hash).await;
+        assert_eq!(aborted_blocks, vec![new_block_hash]);
 
-        let first_block_after_abort = &devnet
+        let new_block_after_abort = &devnet
             .send_custom_rpc(
                 "starknet_getBlockWithTxHashes",
                 json!({
-                    "block_id": {"block_hash": to_hex_felt(&first_block_hash)},
+                    "block_id": {"block_hash": to_hex_felt(&genesis_block_hash)},
                 }),
             )
             .await["result"];
-        assert_eq!(first_block_after_abort["status"], "ACCEPTED_ON_L2".to_string());
+        assert_eq!(new_block_after_abort["status"], "ACCEPTED_ON_L2".to_string());
```

Comment:
What is the purpose of this check? To assert that the genesis block is still accepted? If so, can you add a one-line comment explaining that?

Reply:
"genesis block is still accepted?" -> yes I'll rename this variable and add comment